org.osgi.test.junit5/src/test/java/org/osgi/test/junit5/bundle/BundleArgumentProviderTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) OSGi Alliance (2020). All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.osgi.test.junit5.bundle;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.osgi.framework.Bundle;
+import org.osgi.test.assertj.bundle.BundleAssert;
+import org.osgi.test.assertj.dictionary.DictionaryAssert;
+
+public class BundleArgumentProviderTest {
+
+	@ParameterizedTest
+	@BundleSource(stateFilter = Bundle.ACTIVE)
+	public void testParam(Bundle bundle) throws Exception {
+		BundleAssert.assertThat(bundle)
+			.isInState(Bundle.ACTIVE);
+	}
+
+	@ParameterizedTest
+	@BundleSource(headerFilter = "(Bundle-Vendor=OSGi Alliance)")
+	public void testHeader(Bundle bundle) throws Exception {
+		DictionaryAssert.assertThat(bundle.getHeaders())
+			.containsEntry("Bundle-Vendor", "OSGi Alliance");
+	}
+
```

Comment:
No test for symbolic name filter?

Reply:
The code coverage report highlights this lack of test coverage too.

Reply:
added a test an will look to the coverage